renderdoc/driver/gl/gl_common.cpp

```diff
@@ -2203,7 +2204,16 @@ void CopyProgramFragDataBindings(const GLHookSet &gl, GLuint progsrc, GLuint pro
 
     GLint idx = gl.glGetFragDataLocation(progsrc, refl->OutputSig[i].varName.elems);
     if(idx >= 0)
-      gl.glBindFragDataLocation(progdst, (GLuint)idx, refl->OutputSig[i].varName.elems);
+    {
+      if(gl.glBindFragDataLocation)
+        gl.glBindFragDataLocation(progdst, (GLuint)idx, refl->OutputSig[i].varName.elems);
```

Comment:
Can you put {}s around the if here? I find it's easier to read the if and else as matching if they're both bracketed or both bare.